gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Can npm run scripts allow you to debug your spec files? If so, that is great. Otherwise, the only purpose for the gulp is for debugging unit tests and I can keep it out of the main repo.

Although, excluding it would make portability an issue across dev machines.